xarray/core/dataset.py

```diff
@@ -2742,6 +2826,42 @@ def head(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Examples
+        --------
+        >>> dates = pd.date_range(start="2023-01-01", periods=5)
+        >>> pageviews = [1200, 1500, 900, 1800, 2000]
+        >>> visitors = [800, 1000, 600, 1200, 1500]
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "pageviews": (("date"), pageviews),
+        ...         "visitors": (("date"), visitors),
+        ...     },
+        ...     coords={"date": dates},
+        ... )
+
+        # Retrieve the 3 most busiest days in terms of pageviews and visitors
+
+        >>> busiest_days = dataset.sortby(["pageviews", "visitors"], ascending=False)
```

Comment:
```suggestion
        >>> busiest_days = dataset.sortby("pageviews", ascending=False)
```
If my understanding is correct then in this case sorting by both pageviews and visitors just gives the same result as sorting by visitors, but in a less clear way.